examples/bri-3/src/bri/identity/bpiSubjects/capabilities/getAllBpiSubjects/getAllBpiSubjectsQuery.handler.ts

```diff
@@ -1,24 +1,22 @@
 import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
-import { BpiSubjectDto } from '../../api/dtos/response/bpiSubject.dto';
 import { BpiSubjectStorageAgent } from '../../agents/bpiSubjectsStorage.agent';
 import { GetAllBpiSubjectsQuery } from './getAllBpiSubjects.query';
-import { InjectMapper } from '@automapper/nestjs';
 import { BpiSubject } from '../../models/bpiSubject';
-import { Mapper } from '@automapper/core';
+import { PrismaMapper as Mapper } from '../../../../../shared/prisma/prisma.mapper';
 
 @QueryHandler(GetAllBpiSubjectsQuery)
 export class GetAllBpiSubjectsQueryHandler
   implements IQueryHandler<GetAllBpiSubjectsQuery>
 {
   constructor(
-    @InjectMapper() private autoMapper: Mapper,
+    private readonly mapper: Mapper,
```

Comment:
Please revert this, Prisma mapper should be used only in the storage level, so between prisma models and domain object

Reply:
Done.